sonic_installer/main.py

```diff
@@ -480,7 +480,9 @@ def sonic_installer():
 @sonic_installer.command('install')
 @click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
               expose_value=False, prompt='New image will be installed, continue?')
-@click.option('-f', '--force', is_flag=True,
+@click.option('--force_secureboot', is_flag=True,
+              help="Force installation of an image of a type which is not secureboot")
+@click.option('-f', '--force_install', is_flag=True,
```

Comment:
>'-f', '--force_install' [](http://example.com/codeflow?start=14&length=23)

How about '--skip-platform-check'. Since this one is not used much, it's okay to have only long option.

Reply:
Use dash instead of underscore in option name.